spec/03-plugins/08-datadog/01-log_spec.lua

```diff
@@ -205,6 +210,14 @@ for _, strategy in helpers.each_strategy() do
         },
       }
 
+      bp.plugins:insert {
+        name     = "datadog",
+        route = { id = route8.id },
+        config   = {
+          host             = "{vault://env/kong-datadog-agent-host}",
```

Comment:
I feel we can use a different env var, so that if we accidently run into the code path that "use the env var when `host` is unset", it will help us catch the regression